src/util/apiClientHelper.ts

```diff
@@ -0,0 +1,26 @@
+import moment from 'moment';
+
+export default class ApiClientHelper {
+    private static arrayToObject = (arr: [string, any][]) => {
+        const res = {};
+        arr.forEach(([key, value]:[string, any]) => {
+            res[key] = value;
+        });
+        return res;
+    }
+
+    public static format = (obj: object) => {
+        // Formats the object to include dates as formatted strings, instead of Date objects
+        const entries = Object.entries(obj);
+        const dates = entries
+            .filter(([key, value]: [string, any]) => value instanceof Date)
+            .map(
+                ([key, value]: [string, any]) =>
```

Comment:
We just filtered out anything where the value isn't Date, so make this `[string, Date]` ?

Reply:
true